Dockstore.cwl

```diff
@@ -17,7 +17,7 @@ dct:contributor:
 
 requirements:
 - class: DockerRequirement
-  dockerPull: quay.io/pancancer/pcawg-sanger-cgp-workflow:2.0.3
+  dockerPull: quay.io/pancancer/pcawg-sanger-cgp-workflow:expose-cpu-memory
```

Comment:
Is this correct? 
Maybe something like `quay.io/pancancer/pcawg-sanger-cgp-workflow:2.0.3-expose-cpu-memory` would make it easier to tell where this falls in terms of versions

Reply:
Good question, this should be a general thing, this sounds like a good practice we should follow.
Just to be clear, currently, we just make docker image tag to match git repo branch name. When formally making a release, we will need to update docker image tag to match the release tag.